k8s/cassandra/apptest/tester/tests/basic-suite.yaml

```diff
@@ -20,9 +20,9 @@ actions:
       }
 
       function wait_for_healthy_sts {
-        info "Waiting for equal desired and current number of replicas"
+        stderr_msg "Waiting for equal desired and current number of replicas"
         while [[ $(get_current_number_of_replicas_in_sts) -ne $(get_desired_number_of_replicas_in_sts) ]]; do
-          info "Sleeping 10 seconds before rechecking..."
+          stderr_msg "Sleeping 10 seconds before rechecking..."
           sleep 10
         done
         info "Statefulset has equal current and desired number of replicas"
```

Comment:
```suggestion
        stderr_msg "Statefulset has equal current and desired number of replicas"
```

Reply:
Not needed, will be removed, please check previous comment.